build/run.js

```diff
@@ -127,7 +127,7 @@ commands.build.rust = async function(argv) {
 
         console.log('Checking the resulting WASM size.')
         let stats = fss.statSync(paths.dist.wasm.mainOptGz)
-        let limit = 3.54
+        let limit = 3.56
```

Comment:
where this increase comes from? I'd understand 0.01 MB, but 0.02 MB seems big for these changes.

Reply:
IDK, but maybe it's autogenerated code from serde macros?

Reply:
I'm wondering it Michal did not increase it more than needed :P Because it seems like a lot.